### PR TITLE
Order route created

### PR DESCRIPTION
**Pull request by JefValenco**
- Order route created

### CODE REVIEW

File: src/controllers/Order.controller.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      totalAmount,
+      itemName,
+      itemQuantity,
+      itemPrice,
+      orderDate,
+    });
+
+    res.status(201).json(newOrder);
+  } catch (error) {
+    console.log(req.body);
```
Comment (Mbarca89) [inline src/controllers/Order.controller.js:25]: delete console.log

File: src/models/Order.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { Schema, model } from "mongoose";
+
+const conectionString =
+  "mongodb+srv://pftestuser:dCIJSIvnkiwSUaug@cluster0.rzccgck.mongodb.net/?retryWrites=true&w=majority";
```
Comment (Mbarca89) [inline src/models/Order.js:4]: delete conetionString

File: src/models/Order.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+const orderSchema = new Schema(
+  {
+    userID: {
```
Comment (Mbarca89) [inline src/models/Order.js:8]: userId should be taken as reference from User collection

File: src/models/Order.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      type: String,
+      required: true,
+    },
+    totalAmount: {
+      type: Number,
+      required: true,
+    },
+
+    itemName: {
```
Comment (Mbarca89) [inline src/models/Order.js:17]: itemName, itemQuantity and itemPrice should be replaced by the same products array from Cart schema using reference

**Review by Mbarca89 — CHANGES_REQUESTED** [review]
apply modifications in comments and take a look at the new Cart schema

**Comment by xMaxiuxx** [comment]
in spanish pls 😄

**Pull request closed without merging**